mappings/net/minecraft/entity/ai/brain/task/SniffTask.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_rgxeulpd net/minecraft/entity/ai/brain/task/SniffTask
+	FIELD f_mjeinqgz MAX_ANGER_DISTANCE_VERTICAL D
+	FIELD f_mjlpaudz MAX_ANGER_DISTANCE_HORIZONTAL D
```

Comment:
What does this mean?

Reply:
The maximum horizontal and vertical distances from the Warden entity that will anger it